spidermon/utils/web.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+if sys.version_info[0] == 3:
+    from urllib.parse import urlparse
+    from urllib.request import urlopen
+else:
+    from urlparse import urlparse
+    from urllib import urlopen
+
+
+def is_schema_url(path):
+    result = urlparse(path)
+    try:
+        if all([result.scheme, result.netloc, result.path]):
+            return True
+        return False
+    except AttributeError:
+        return False
+
+
+def get_contents(url):
+    with urlopen(url) as f:
```

Comment:
Same as https://github.com/scrapinghub/spidermon/pull/119/files#r263525248, you could be more friendly to the user handling `URLError` and logging the reason for the failure.